resources/integration_test/tests/columns_no_filter.yml

```diff
@@ -0,0 +1,251 @@
+tests:
+  - description: SQLColumnsW no filters
+    test_definition: [ "sqlcolumnsw", "", 0, "", 0, "", 0, "", 0 ]
+    db: integration_test
+    expected_result:
+      - ["integration_test", null, "example", "_id", 4, "int", 10, 4, 0, 10, 1, "", null, "{\"$numberInt\":\"4\"}" , null , 0, 1, "YES"]
+      - ["integration_test", null, "example", "b", 12, "string", 0, 4294967292, 0, null, 1, "", null, 12, null, 4294967292, 2, "YES"]
+      - ["integration_test", null, "foo", "_id", 4, "int", 10, 4, 0, 10, 1, "", null, "{\"$numberInt\":\"4\"}" , null , 0, 1, "YES"]
+      - ["integration_test", null, "foo", "a", 4294967291, "long", 19, 8, 0, null, 1, "", null, 4294967291, null, 0, 2, "YES"]
+      - ["integration_test_2", null, "example_2", "_id", 8, "double", 15, 8, 15, 10, 1, "", null, 8 , null , 0, 1, "YES"]
+      - ["integration_test_2", null, "example_2", "b", 12, "string", 0, 4294967292, 0, null, 1, "", null, 12, null, 4294967292, 2, "YES"]
+    expected_bson_type: [
+        "string",
```

Comment:
Looks like the output here comes from the test generator, do you know if there is a way to format the YAML output to have these lists in one line instead of a newline for each entry?  I checked briefly at `serde_yaml::to_writer()` and couldn't find a way, but it would make the file more readable.  No worries if it's not a quick fix.

Reply:
One other suggestion is if the metadata for all the SQLColumnsW calls are the same, we can do like the JDBC tests and have only one of the tests check all the metadata.  The remaining `SQLColumnsW` will then only need to check that the results are correct (and skip the metadata checking).  This would be clearer that the metadata is checked by one of the tests if these were in one SQLColumnsW yaml file as Natacha suggests.

Reply:
I made these manually. I didn't want them on one line.

Reply:
Oh, I don't have to check the metadata! Yes!